lib/bios/menus.robot

```diff
@@ -59,6 +59,25 @@ Enter Dasharo System Features Submenu Snapshot
     ${menu_construction}=    Get Setup Submenu Construction
     RETURN    ${menu_construction}
 
+Enter Dasharo Submenu
```

Comment:
@Psotas We cannot accept adding new keywords that provide redundant functionality anymore. This is the exact approach that was used here in this repo, that led to the current state that we have 8 keywords for logging into operating system...

If you opt for improvement of the keyword, you shall improve/replace the existing one, and all of it's occurrences. If this is too much, create an issue describing your proposal, but in these MR you should use existing ones.

Reply:
OK, I will try to improve the keywords.
I was planning to do it in a separate PR, now I know to do it in the same one.